rdrf/rdrf/views/session_refresh_view.py

```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
+from django.http import JsonResponse
+from django.views.decorators.http import require_GET
+
+
+@require_GET
+def session_refresh(request):
```

Comment:
`num_retries` only ever goes down. Should be reset on page loads

Reply:
Updated.